src/sentry/tasks/derive_code_mappings.py

```diff
@@ -20,7 +20,7 @@
 ACTIVE_PROJECT_THRESHOLD = timedelta(days=7)
 GROUP_ANALYSIS_RANGE = timedelta(days=14)
 
-logger = logging.getLogger("sentry.tasks.derive_code_mappings")
+logger = logging.getLogger(__name__)
```

Comment:
This is the same value. If we move the file we will not need to remember to update this.